<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget #Form {
background-color: rgb(230, 230, 230);
}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="text">
    <string>Image Rotator</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="step_one">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="title">
    <string>Step 1 - Select Image Files</string>
   </property>
   <widget class="QPushButton" name="button_choose_files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Select Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_no_files">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0 files chosen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step_two">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="title">
    <string>Step 2 - Choose Rotation</string>
   </property>
   <widget class="QWidget" name="ninety_clockwise" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #ninety_clockwise {
	border-radius : 7px;
	border-width : 1px;
	border-style : solid;
	border-color: rgb(0, 0, 0);
}</string>
    </property>
    <widget class="QLabel" name="label_ninety_clockwise_ninety">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>90°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ninety_clockwise_clockwise">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Gill Sans MT&quot;;</string>
     </property>
     <property name="text">
      <string>clockwise</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ninety_clockwise_arrow">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>2</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>clockwise.JPG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ninety_counterclockwise" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #ninety_counterclockwise {
	border-radius : 7px;
	border-width : 1px;
	border-style : solid;
	border-color: rgb(0, 0, 0);
}</string>
    </property>
    <widget class="QLabel" name="label_ninety_counterclockwise_ninety">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>90°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ninety_counterclockwise_clockwise">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Gill Sans MT&quot;;</string>
     </property>
     <property name="text">
      <string>c-clockwise</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ninety_counterclockwise_arrow">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>2</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>counter-clockwise.JPG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="onehundredeighty" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #onehundredeighty {
	border-radius : 7px;
	border-width : 1px;
	border-style : solid;
	border-color: rgb(0, 0, 0);
}</string>
    </property>
    <widget class="QLabel" name="label_onehundredeighty_angle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>180°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_onehundredeighty_arrow">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>2</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>half-rotation.JPG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="custom" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #custom {
	border-radius : 7px;
	border-width : 1px;
	border-style : solid;
	border-color: rgb(0, 0, 0);
}</string>
    </property>
    <widget class="QLabel" name="label_custom">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="clockwise">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Clockwise</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="counter_clockwise">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C-Clockwise</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="angle">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step_four">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="title">
    <string>Step 4 - Saving Image Files</string>
   </property>
   <widget class="QPushButton" name="button_choose_destination">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Select Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_destination">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;location&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(85, 255, 255);
font: 14pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_success">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;
color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Error / Success</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="text">
    <string>Created by: Nicholas Williams</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="step_three">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
   </property>
   <property name="title">
    <string>Step 3 - Choose Quality/File Size Reduction</string>
   </property>
   <widget class="QPushButton" name="button_same">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Same Quality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_different">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(85, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Different Quality</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="percentage_quality">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>75</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_percentage">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Gill Sans MT&quot;;</string>
    </property>
    <property name="text">
     <string>% of Original</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
